content/03-type-system.md

```diff
@@ -155,16 +155,16 @@ Main.main();
 >
 > A type parameter is said to be generic if its containing class or method is generic.
 
-It is not possible to construct normal type parameters, e.g. `new T()` is a compiler error. The reason for this is that Haxe generates only a single function and the construct makes no sense in that case. This is different when the type parameter is generic: Since we know that the compiler will generate a distinct function for each type parameter combination, it is possible to replace the `T` `new T()` with the real type.
+It is not possible to construct normal type parameters; for example, `new T()` would register as a compiler error. The reason for this is that Haxe generates only a single function and the construct would make no sense in that case. This is different when the type parameter is generic: since we know that the compiler will generate a distinct function for each type parameter combination, it is possible to replace the `T` `new T()` with the real type.
 
 [code asset](assets/GenericTypeParameter.hx)
 
-It should be noted that [top-down inference](type-system-top-down-inference) is used here to determine the actual type of `T`. There are two requirements for this kind of type parameter construction to work: The constructed type parameter must be
+It should be noted that [top-down inference](type-system-top-down-inference) is used here to determine the actual type of `T`. There are two requirements for this kind of type parameter construction to work: The constructed type parameter must be:
```

Comment:
In other places you're changing it to lower-case-after-colon, so I suppose we should do this here too. I don't like this double-colon in general though, so maybe this can be rephrased differently altogether.

Reply:
It turns out that capitalization after colons can go either way, which I was unaware of (sorry). If desired I can shift it back to the way it was before, no big deal really.

I'll try and rephrase this section.